test/Sema/where_clause_across_module_boundaries.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -o %t/ModuleA.swiftmodule %S/Inputs/where_clause_across_module_boundaries_module.swift
+// RUN: %target-swift-frontend -I %t -emit-sil -verify %s
```

Comment:
Although there is no formal convention, it is preferable to align the compilation mode with the scope of the test, in this case `-typecheck` (or `%target-typecheck-verify-swift`) rather than `-emit-sil`.

Reply:
Got it! Thanks for the review!